notebooks/HATS/Exercise 6 - JECNtuple\ Setup.ipynb

```diff
@@ -0,0 +1 @@
+../master/JECNtuple Setup.ipynb
```

Comment:
Shouldn't you also then rename the links `
Exercise 6 - Jet Energy Corrections - V2.ipynb` and `Exercise 6 - Jet Energy Corrections.ipynb` to be exercise 7? Pushing all of the higher numbers up by 1.